ci/build_container/build_recipes/protobuf.sh

```diff
@@ -10,3 +10,10 @@ rsync -av protobuf-$VERSION/* $THIRDPARTY_SRC/protobuf
 cd protobuf-$VERSION
 ./configure --prefix=$THIRDPARTY_BUILD --enable-shared=no
 make V=1 install
+
+# These internal headers are needed by the GRPC-JSON transcoding library.
+# https://github.com/grpc-ecosystem/grpc-httpjson-transcoding
+# TODO(htuch): Clean up protobuf deps builds with envoy-api
+rsync -av --include '*/' --include '*.h' --exclude '*' src/google/protobuf/util/internal $THIRDPARTY_BUILD/include/google/protobuf/util/
```

Comment:
This could probably be a simpler src/google/protobuf/util/internal/*.h expression...